pkgs/development/python-modules/ovirt-engine-sdk/default.nix

```
@@ -0,0 +1,37 @@
+{ 
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pycurl,
+  six,
+  libxml2Python
+}:
+
+buildPythonPackage rec {
+  pname = "ovirt-engine-sdk-python";
+  version = "4.3.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0hz7rcz7l391n8ixcwvqw3iq5bx26fk9l5c6xxymv036zm3r9pnh";
+  };
+
+  doCheck = true;
+
+  buildInputs = [ 
```

for python packages that are needed at runtime, you should use propagatedBuildInputs
```suggestion
  propagatedBuildInputs = [ 
```

---

fixed